src/rtpghi~.c

```diff
@@ -56,10 +73,13 @@ t_int *rtpghi_tilde_perform(t_int *w) {
 
 void rtpghi_tilde_dsp(t_rtpghi_tilde *x, t_signal **sp)
 {
-	post("start DSP");
+	//post("start DSP");
     ltfat_int M = (ltfat_int) sp[0]->s_n;
+	if ((x->blocksize) == 0) {
+		(x->blocksize) = M;
```

Comment:
very minor nitpick, but what is it with your indentation?
is it just your editor that handles tabs and spaces separately (with a tab having the same indentation as 4 spaces), or is there some hidden meaning?

it's driving me crazy :-/